crates/viewer/re_view_spatial/src/mesh_loader.rs

```diff
@@ -1,14 +1,18 @@
 use itertools::Itertools;
 use re_chunk_store::RowId;
 use re_renderer::{mesh::GpuMesh, RenderContext, Rgba32Unmul};
-use re_types::{
-    archetypes::{Asset3D, Mesh3D},
-    components::{AlbedoFactor, MediaType},
-};
+use re_types::{archetypes::Mesh3D, components::MediaType};
 use re_viewer_context::{gpu_bridge::texture_creation_desc_from_color_image, ImageInfo};
 
 use crate::mesh_cache::AnyMesh;
 
+#[derive(Debug, Clone)]
+pub struct Asset3D<'a> {
```

Comment:
Can we name that `NativeAsset3D` or something? This will be quite confusing in the future.

Reply:
`NativeAsset3D` sounds good to me